salt/cloud/clouds/ec2.py

```diff
@@ -4205,7 +4205,7 @@ def show_keypair(kwargs=None, call=None):
         return False
 
     params = {'Action': 'DescribeKeyPairs',
-              'KeyName.1': kwargs['keyname']}
```

Comment:
`DescribeKeyPairs` is still `KeyName.1` according to http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeKeyPairs.html